framework/integration/init.go

```diff
@@ -17,38 +17,13 @@ import (
 	"github.com/giantswarm/operatorkit/informer"
 )
 
-const (
-	namespace = "finalizer-integration-test"
-)
-
 var (
 	err error
 
 	k8sClient kubernetes.Interface
 )
 
-func init() {
```

Comment:
Just some reordering after I made functions needed in tests public.

Again might move this into another folder with a package called `setup` to align with aws-operator but not sure if that makes sense in this case.